docker-compose.yml

```diff
@@ -46,9 +46,11 @@ services:
       - RELAYER_CERT_HOST=${RELAYER_CERT_HOST}
       - RELAYER_CERT_PATH=/secure/relayer-root.pem
       # WARNING: Only disable broker SSL during development
-      - DISABLE_AUTH=false
+      - DISABLE_AUTH=true
```

Comment:
DISABLE_AUTH used to control authentication to the Relayer, right? Has that been changed to a different environment variable? And why is this disabled by default?

Reply:
DISABLE_AUTH is now for broker auth where as DISABLE_RELAYER_AUTH is used for the relayer value

Reply:
Cool - why is broker auth disabled by default? Is it a "until we get docs in place" thing?

Reply:
@treygriffith until its an official release